app/controllers/settings_controller.py

```diff
@@ -422,6 +421,9 @@ def _update_view_from_model(self) -> None:
             self.settings.external_steam_metadata_repo
         )
         self.settings_dialog.steam_workshop_db_github_url.setCursorPosition(0)
+        self.settings_dialog.db_builder_database_expiry.setText(
```

Comment:
this should probably be renamed for clarity

Reply:
ok done should be good now